home-application/src/main/java/com/softserveinc/ita/homeproject/application/config/StringToEnumConverter.java

```diff
@@ -0,0 +1,17 @@
+package com.softserveinc.ita.homeproject.application.config;
+
+import javax.ws.rs.ext.Provider;
+
+import com.softserveinc.ita.homeproject.homedata.entity.ContactType;
+import org.springframework.core.convert.converter.Converter;
+
+
+
+
+@Provider
+public class StringToEnumConverter implements Converter<String, ContactType> {
```

Comment:
As we stuck to one particular type (which I think isn't scalable enough) the name of the class must be more precise (e.g. ContactTypeEnumConverter). Consider renaming the class or making it more flexible.